### DISCUSSION TITLE
Is there any "Buy me a coffe"-sort-of link?

### DISCUSSION THREAD

**Discussion by raquelhortab** [optuna/optuna#4651] [Q&A] [answered]
I am currently doing a uni project and I am quite pleased with this tool, altogether with the dashboard package. So I'd like to thank the developers and I was wondering if there is  any donation link, kind of like [buymeacoffee.com](https://www.buymeacoffee.com/) :smile:

**Comment by nzw0301** [comment, marked as answer]
@raquelhortab FYI https://github.com/optuna/optuna/issues/4736#issuecomment-1710997122: Optuna starts to accept GitHub sponsors.

**Comment by raquelhortab** [comment]
Amazing! I've just donated a bit! thanks for telling me :)
